sbt-reladomo-plugin/src/test/resources/reladomo/issue009/ReladomoClassList.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<!--
+  Copyright 2017 Goldman Sachs.
```

Comment:
@itohiro73 What should I write here?

Reply:
This is totally a brand new file and you should put "Copyright 2017 FOLIO Co., Ltd." there.